src/main/java/com/hubspot/jinjava/JinjavaConfig.java

```diff
@@ -43,17 +43,18 @@
 
   private Map<Context.Library, Set<String>> disabled;
   private final boolean failOnUnknownTokens;
+  private final boolean noNestedInterpretation;
```

Comment:
in general, I think it's easier to grok booleans as positives rather than negatives. So instead of `noNestedIntepretation`, maybe `enableNestedExpressions`